.zenodo.json

```
@@ -0,0 +1,51 @@
+{
+    "license": "MIT",
+    "upload_type": "software",
+    "creators": [
+        {
+            "name": "Mark Howells",
+            "orcid": "0000-0001-6419-4957",
+            "affiliation": 	"Imperial College London; Loughborough University Department of Social Sciences"
+        },
+        {
```

```suggestion
        {
           "name": "Manuel Welsch",
           "affiliation": "International Atomic Energy Agency"
           },
         {
           "name": "Constantinos Taliotis",
           "orcid": "0000-0003-4022-5506",
           "affiliation": "The Cyprus Institute"
           },
         {
           "name": "Adrian Lefvert",
           "orcid": "0000-0001-8587-4054",
           "affiliation": "KTH Royal Institute of Technology"
           },
         {
           "name": "Igor Tatarewicz",
           "orcid": "",
           "affiliation": ""
           }
         {
           "name": "Tom Alfstad",
            "affiliation": "United Nations Department of Economic and Social Affairs"
            }
          {
            "name": "Nawfal Saadi",
            "orcid": "0000-0001-8923-7431",
            "affiliation": "Enel Green Power"
            }
```

Missing the current affiliation for Tatarewicz.

---

@willu47 I'm not sure of the contribution that is required to be added as author. I've not contributed to the OSeMOSYS code, but added pre-processing scripts that I've used in my work. So I'm not sure I qualify as author here.
@tniet I agree that the order of contribution is important and that it would be good to make the order in the terms of contribution.

---

Contribution to anything in the repository - particularly the model code and the contents of the scripts folder - qualifies you as an author.